src/deprecates.jl

```diff
@@ -48,3 +48,15 @@ findat(a::AbstractArray, b::AbstractArray) = findat!(Array{Int}(size(b)), a, b)
 @deprecate df_residual(obj::StatisticalModel) dof_residual(obj)
 
 @deprecate_binding WeightVec Weights
+
+using Base.Random: RangeGenerator
+@deprecate_binding RandIntSampler RangeGenerator
+@deprecate RangeGenerator(K::Int) RangeGenerator(1:K)
```

Comment:
Ah, okay. Well, not ideal, and we might get some pushback for the type piracy, but idk what else to suggest and I'm sure you've already carefully considered anything else that I could suggest.